README.md

```diff
@@ -106,22 +106,30 @@ git --version
     ```sh
     arduino-cli core install rp2040:rp2040@3.9.5
     ```
-3. Install additional arduino-cli dependencies:
+1. Install additional arduino-cli dependencies:
     ```sh
     arduino-cli lib install Time
     ```
     ```sh
     arduino-cli lib install RadioHead
     ```
+1. Generate the initial fprime build cache
+   ```sh
+   fprime-util generate
+   ```
 
 ### Deploy onto the RP2040
 1. Build the binary
    ```sh
-   fprime-util generate && fprime-util build -j20
+   fprime-util build -j20
+   ```
+
+1. Upload the firmware to the proves board over USB
+   ```sh
+   arduino-cli upload -v -b 115200 --fqbn rp2040:rp2040:rpipico -p /dev/ttyACM0 -i build-artifacts/rpipico/BroncoDeployment/bin/BroncoDeployment.uf2
    ```
```

Comment:
I think it would be a good idea to add here a little note that reminds people to change the `/dev/ttyACM0` to whatever their local path for the target board is. 

I would think something like this would make sense: 
> Make sure to change `/dev/ttyACOM` to the correct path for your target board. You may want to run `ls /dev/tyy*` for a list of all possible paths.

Reply:
What do you think about [422a836](https://github.com/proveskit/fprime-proves/pull/39/commits/422a8367f0b261b1d8a0b08f6bb343af4f8597b1)?

Reply:
Yeah these changes look good to me! I just added some comments for 2 typos that should be corrected